src/components/14-alert/alerts.njk

```diff
@@ -1,35 +1,100 @@
-<h3>Warning</h3>
-<div class="sds-alert">
-  <i class="sds fa-exclamation fa-2x"></i>
-  <span class="sds-alert-content">
-    <b>
-      <i>Address Needs to be Updated</i>
-    </b><br>
-    5438 Kerluke Lake Apt. 937<br>
-    Glen Burnie, MD 21061 United States
-  </span>
-</div>
+<h3>Desktop</h3>
 
-<h3>Info</h3>
 <div class="sds-alert-info">
-  <i class="sds fa-exclamation fa-2x"></i>
-  <span class="sds-alert-content">
-    <b>
-      <i>Address Needs to be Updated</i>
-    </b><br>
-    5438 Kerluke Lake Apt. 937<br>
-    Glen Burnie, MD 21061 United States
-  </span>
+  <div class="grid-container width-full maxw-full margin-0 padding-0">
+    <div class="grid-row">
+      <div class="grid-col-auto">
+        <i class="sds fa-alert-info fa-2x"></i>
+      </div>
+      <div class="grid-col-fill">
+        <span class="sds-alert-content">
+          <b>Less important than Global Alerts</b> Consectetur adipiscing elit, sed do eiusmod tempor indididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.
+        </span>
+      </div>
+      <div class="grid-col-auto">
+        <div class="sds-alert-button">
+          <button aria-controls="alert-info-extension" class="accordion simple-toggle usa-button--circle usa-button--secondary usa-button--secondary-circle search-button--secondary margin-0">
+            <i class="fa fa-caret-down"></i>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="grid-row" id="alert-info-extension" style="display:none;">
```

Comment:
Sorry, we should change this to class display-none (everywhere), I had added this to test only.